src/fs/file.rs

```diff
@@ -420,6 +465,21 @@ impl<'dir> File<'dir> {
         }
     }
 
+    #[cfg(windows)]
+    pub fn attributes(&self) -> f::Attributes {
+        let bits = self.metadata.file_attributes();
+        let has_bit = |bit| bits & bit == bit;
+
+        f::Attributes {
```

Comment:
The constants are grabbed from MSDN doc [here](https://docs.microsoft.com/en-us/windows/win32/fileio/file-attribute-constants).

Reply:
Could this piece of information and MSDN link be added to the source code, as a comment?